# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/voice_engine/channel.h
# Commit: 0c885324ba6f
# Full Hash: 0c885324ba6f42ddb74fe60aca6e2351a3fa88cf
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-09 10:01:40
# Regressor Bug: 1376873
# File Overlap Count: 1
# Description:
#   Bug 1376873 - Fix GetRTPStatistics; r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D7471
# ==============================================================================

diff -r cb6c555654c0 -r 0c885324ba6f media/webrtc/trunk/webrtc/voice_engine/channel.h
--- a/media/webrtc/trunk/webrtc/voice_engine/channel.h	Mon Aug 13 10:24:46 2018 -0400
+++ b/media/webrtc/trunk/webrtc/voice_engine/channel.h	Thu Sep 06 16:24:33 2018 -0400
@@ -357,6 +357,13 @@
     return rtp_receiver_->GetSources();
   }
 
+  int GetPlayoutFrequency() const {
+    if (audio_coding_) {
+      return audio_coding_->PlayoutFrequency();
+    }
+    return 0;
+  }
+
  private:
   class ProcessAndEncodeAudioTask;
 
